evil-dired.el

```diff
@@ -131,11 +131,11 @@
     "x" 'dired-do-flagged-delete
     "y" 'dired-show-file-type
     "+" 'dired-create-directory
-    ;; goto
-    "o" 'dired-find-file-other-window
-    (kbd "C-o") 'dired-display-file
+    ;; follow
+    (kbd "S-<return>") 'dired-find-file-other-window
```

Comment:
Mentioned above, but it'd be good to have an idea how we want to support terminal usage.

Reply:
In this case, `O` is already taken, but maybe that won't be the case once the dired bindings have been worked out.